packages/synapse-react-client/src/components/Ecosystem/EcosystemLayout.tsx

```diff
@@ -0,0 +1,83 @@
+import { Box, Tab, Tabs, useMediaQuery, useTheme } from '@mui/material'
+import Fade from '@mui/material/Fade'
+import { ReactNode, useState } from 'react'
+
+type Config = {
+  title: ReactNode
+  content: ReactNode
+}
+
+export type EcosystemProps = {
+  config: Config[]
+}
+
+/**
+ * Layout for the Ecosystem component, which renders tab buttons on the left side and content on the right side.
+ * @param props
+ * @constructor
+ */
+const EcosystemLayout = (props: EcosystemProps) => {
```

Comment:
Original `Ecosystem` component was 
1. Moved to SRC
2. Renamed to EcosystemLayout
3. Refactored to accept any content instead of just Markdown parameters
4. Enhanced to support a different layout at narrow/mobile screen widths